src/api/lib/tasks/dev.rake

```diff
@@ -276,7 +276,9 @@ namespace :dev do
       admin = User.get_default_admin
       User.session = admin
 
-      interconnect = create(:project, name: 'openSUSE.org', remoteurl: 'https://api.opensuse.org/public')
+      interconnect = create(:remote_project, name: 'openSUSE.org', remoteurl: 'https://api.opensuse.org/public')
+      # The interconnect doesn't work unless we set the distributions
+      FetchRemoteDistributionsJob.perform_now
```

Comment:
I'm trying to create the distribution(s) using factories instead of calling the job. Stil trying to make it build when the distribution is there.

Reply:
In the end, we decided to leave the call to the job.